src/main/scala/org/broadinstitute/hail/variant/Genotype.scala

```diff
@@ -201,6 +201,13 @@ class Genotype(private val _gt: Int,
       b.append(pl.map(_.mkString(",")).getOrElse("."))
     else
       b.append(dosage.map(_.mkString(",")).getOrElse("."))
+
+    val regex = """(?::\.)+$""".r
+    val regexMatch = regex.findFirstMatchIn(b)
+    if (regexMatch.isDefined) {
+      b.delete(regexMatch.get.start, regexMatch.get.end)
+    }
```

Comment:
Thanks Cotton, that is done. Thanks again for the great starter descriptions. I'll look around for something else to poke around in.